airflow/migrations/versions/0147_2_10_0_rename_dagrun_last_scheduling_decision.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Rename dagrun.last_scheduling_decision.
+
+Revision ID: 840479e61d99
+Revises: d482b7261ff9
+Create Date: 2024-06-18 08:58:10.356565
+
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+
+from airflow.utils import timezone
+from airflow.utils.sqlalchemy import UtcDateTime
+
+# revision identifiers, used by Alembic.
+revision = "840479e61d99"
+down_revision = "d482b7261ff9"
+branch_labels = None
+depends_on = None
+airflow_version = "2.10.0"
+
+
+def upgrade():
+    """Apply Rename dagrun.last_scheduling_decision."""
+    with op.batch_alter_table("dag_run", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("next_schedulable", UtcDateTime(), default=timezone.utcnow()))
```

Comment:
Have you tested how long this takes to apply with a large table?

Reply:
I have tested the migration with 5.4 million dagrun rows and it took 6s+
<img width="544" alt="Screenshot 2024-06-24 at 18 47 55" src="https://github.com/apache/airflow/assets/4122866/8b0183bd-811d-4ae1-8ba0-e03dc3cad6aa">